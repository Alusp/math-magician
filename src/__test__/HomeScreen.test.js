import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from '../pages/HomeScreen';
import '@testing-library/jest-dom';

describe('Test HomeScreen', () => {
  test('Testing HomeScreen', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
