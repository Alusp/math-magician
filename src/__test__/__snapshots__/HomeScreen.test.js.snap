// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HomeScreen Testing HomeScreen 1`] = `
<div
  className="container mx-auto py-16 px-32"
>
  <h1
    className="text-2xl font-semibold mb-4"
  >
     Welcome to our Page
  </h1>
  <p
    className="text-gray-600"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s  with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
  <p
    className=" mt-3 text-gray-600"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry  standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
</div>
`;
